<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formimageeditor</class>
 <widget class="QWidget" name="Formimageeditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="chkGrid">
         <property name="text">
          <string>Grid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPosition">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Toolbox</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>151</width>
           <height>571</height>
          </rect>
         </property>
         <property name="title">
          <string>Toolbox</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>131</width>
            <height>541</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="lyToolbox_3"/>
         </widget>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>220</y>
           <width>195</width>
           <height>205</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0">
          <item row="5" column="0">
           <widget class="QPushButton" name="btnExportAsm">
            <property name="text">
             <string>Export Asm</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="btnExportImage">
            <property name="text">
             <string>Export image</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnLoad">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btnSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="b">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnImport">
            <property name="text">
             <string>Import image</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="btnSaveAs">
            <property name="text">
             <string>Save As</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="btnNew">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btnImportBin">
            <property name="text">
             <string>Import bin</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="btnTiff">
            <property name="text">
             <string>Load Tiff</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnExportBin">
            <property name="text">
             <string>Export bin</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="btnExpChar">
            <property name="text">
             <string>Export Char</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Colors</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>191</width>
           <height>431</height>
          </rect>
         </property>
         <property name="title">
          <string>Colors</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_6">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>71</width>
            <height>401</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="layoutColorsEdit_3"/>
         </widget>
         <widget class="QComboBox" name="cmbBackgroundMain_3">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>40</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="cmbBorderMain_3">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>120</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Background</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>90</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Border</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cmbMC2">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>290</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>260</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Multicolor 2</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>180</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Multicolor 1</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cmbMC1">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>210</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Charmap</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <widget class="QTableWidget" name="lstCharMap"/>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btnLoadCharmap">
             <property name="text">
              <string>Load Charmap</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="3" column="0">
              <widget class="QPushButton" name="btnCharset2x2Repeat">
               <property name="text">
                <string>2x2 charset repeat</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="btnCharset2x2">
               <property name="text">
                <string>2x2 character set</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="btnCharsetFull">
               <property name="text">
                <string>Full character set</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="btnCharset1x1">
               <property name="text">
                <string>1x1 character set</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="btnCharsetCopy">
               <property name="text">
                <string>Copy</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="btnCharsetPaste">
               <property name="text">
                <string>Paste</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Levels</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLevels"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>Data</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QTableWidget" name="tblData">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnResizeData">
           <property name="text">
            <string>Resize data</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblImageName">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Image Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lblImage">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
       <widget class="QLabel" name="lblGrid">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>12</y>
          <width>791</width>
          <height>531</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>lblImage</zorder>
       <zorder>lblGrid</zorder>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
